ReactiveCocoaTests/Swift/SignalProducerLiftingSpec.swift

```diff
@@ -496,6 +496,114 @@ class SignalProducerLiftingSpec: QuickSpec {
 				observer.sendFailed(.Default)
 				expect(error) == TestError.Default
 			}
+
+			it("should collect a certain count of values") {
+				let (original, observer) = SignalProducer<Int, NoError>.buffer(1)
+
+				let producer = original.collect(3)
+
+				var expectedValues = [
+					[1, 2, 3],
+					[4, 5, 6],
+					[7, 8, 9],
+					[0]
+				]
```

Comment:
I deleted that test which I think was redundant since there was another doing practically the same and that one now have guarantees about the timing of the values.